Havit.Blazor.Components.Web.Bootstrap/Forms/HxInputDateRange.cs

```diff
@@ -105,6 +105,7 @@ static HxInputDateRange()
 	/// </summary>
 	[Parameter] public CalendarDateCustomizationProviderDelegate CalendarDateCustomizationProvider { get; set; }
 	protected CalendarDateCustomizationProviderDelegate CalendarDateCustomizationProviderEffective => this.CalendarDateCustomizationProvider ?? this.GetSettings()?.CalendarDateCustomizationProvider ?? GetDefaults().CalendarDateCustomizationProvider;
+	protected TimeProvider TimeProviderEffective => GetSettings()?.TimeProvider ?? GetDefaults().TimeProvider;
```

Comment:
We should probably also add the `[Parameter] public TimeProvider` and `[Inject] protected TimeProviderFromServices` here.